store-collected-images/src/utils/image-store.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
```

Comment:
Since this is really the key function (AIUI) in terms of illustrating the usage of the storage API, it would be great to have some comments.